include/Track.h

```diff
@@ -146,6 +146,8 @@ class TrackContentObject : public Model, public JournallingObject
 		return m_selectViewOnCreate;
 	}
 
+	/// Returns true if and only if a->startPosition() < b->startPosition()
+	static bool comparePosition(const TrackContentObject* a, const TrackContentObject* b);
```

Comment:
We may end up using it elsewhere. Also, it's quite obviously a function related to `TrackContentObject`, so I don't think moving it to a mostly unrelated file is a good idea. Whether `TrackContentObject` should know it's own position is debatable of course, but that's how it is right now.